docs/source/models/intervention_models/lipid_based_nutrient_supplements/index.rst

```diff
@@ -390,16 +413,31 @@ Additionally, as suggested by the observed prevalence ratios from the meta-analy
 
   * - Outcome
     - Effect size measure
+    - Sensitivity analysis 
     - Effect size
     - Note
   * - Moderate (cat2) stunting exposure
     - Prevalence ratio
+    - Standard effects
     - 0.89 (0.86, 0.93), lognormal distribution of uncertainty
     - 
+  * - Moderate (cat2) stunting exposure
+    - Prevalence ratio
+    - Modified effects
+    - 0.83 (0.78, 0.89) in high burden areas, lognormal distribution of uncertainty
+      0.87 (0.77, 0.98) in low burden areas, lognormal distribution of uncertainty
+    - 
   * - Severe (cat1) stunting exposure
     - Prevalence ratio
+    - Modified effects 
```

Comment:
Should this be "Standard effects" instead of "Modified effects" (it looks like this is the effect size that was in the documentation originally)?